docs/design-systems/variables.md

```diff
@@ -1,3 +1,130 @@
+head:
+  - - meta
+    - name: description
+      content: Discover how variables in the bluefox.email design system enable dynamic customization of email components. Define reusable settings for colors, text, images, and more to maintain consistency while allowing flexibility at the project level.
+  - - meta
+    - property: og:title
+      content: Design System Variables | bluefox.email documentation
+  - - meta
+    - property: og:description
+      content: Discover how variables in the bluefox.email design system enable dynamic customization of email components. Define reusable settings for colors, text, images, and more to maintain consistency while allowing flexibility at the project level.
+  - - meta
+    - property: og:image
+      content: https://bluefox.email/assets/docs-share.png
+  - - meta
+    - property: og:url
+      content: https://bluefox.email/docs/projects
+  - - meta
+    - property: og:type
+      content: website
+  - - meta
+    - property: twitter:card
+      content: summary_large_image
+  - - meta
+    - property: twitter:title
+      content: Design System Variables | bluefox.email documentation
+  - - meta
+    - property: twitter:description
+      content: Discover how variables in the bluefox.email design system enable dynamic customization of email components. Define reusable settings for colors, text, images, and more to maintain consistency while allowing flexibility at the project level.
+  - - meta
+    - property: twitter:image
+      content: https://bluefox.email/assets/docs-share.png
+---
+
 # Design system variables
 
-//TODO
+Variables in the design system allow you to define reusable values that can be applied across templates and blocks. They ensure consistency and make it easy to update design elements across multiple emails without manually modifying each template or block.
+
+When a variable is updated, all templates and blocks using that variable will automatically reflect the changes. This allows for efficient design management and helps maintain a unified brand identity across projects.
+
+## Variables page
+The **variables page** is divided into two main sections:
+
+- **Left panel (variable list)**: Displays different types of variables (colors, fonts, images, text, links). Users can create, edit, delete, and manage variables.
+- **Right panel (preview section)**: Shows how the selected variable is applied in templates or blocks by displaying a live example, providing immediate feedback on its effect on designs.
+
+![A screenshot of the design system variables section.](./design-system-variables.webp)
+
+
+## Types of variables
+
+### 1. Color variables
+Color variables store reusable colors, such as primary colors, neutrals, and accent colors. Each variable includes:
+
+- **Color name** (e.g., `color-primary`)
+- **Hex value** (e.g., `#1B1E1F`)
+- **Main variable** (true/false value indicates whether the variable is visible in the design system preview.)
```

Comment:
I would explain, that these variables will be visible in the project creation dialog. The ones that are not main variables can be changed later.